spec/requests/api/instrumentation_notification_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe 'config/initializers/instrumentation_notification.rb' do
```

Comment:
remove this spec file. there is no information anymore about `current_customer_id`. better to add a spec to one of API request spec

Reply:
@gigorok this changes is not safe and after apply this notice almost all test activeadmin feature test is failed. Maybe I suggest to add this line in process action(*args) method

```
return super unless request.path.start_with?('/api/')
```

The problem is next active admin generates controller and we have error that some_page_controller do not have meta method.